cli/src/pcluster/config/cluster_config.py

```diff
@@ -1086,7 +1085,7 @@ class AwsBatchComputeResource(BaseComputeResource):
 
     def __init__(
         self,
-        instance_types: str = None,
+        instance_types: List[str] = None,
```

Comment:
if instance_types is a list we can move it back to `BaseComputeResource`, it was here because it was different.